doc/decisions/3_in_review/change_tracking.md

```diff
@@ -174,13 +174,27 @@ The changetracking plugin needs to export at least functions for the following t
 
 ## Decision
 
+Plugin and application developers can declare that changetracking is required via a contract.
```

Comment:
I don't know if it necessary to enable it via application developers. Please link a use case for this.

What definitely is needed, is that it can be enabled via tooling (and then stays enabled for everyone).